### PR TITLE
chore(devDependencies): Update dev dependencies to latest

### PR DESCRIPTION
**Pull request by nem035**
⬆️ 📦 some dev dep updates 🙂

### CODE REVIEW

**Comment by nem035** [comment]
Oops, completely missed that 🙂 Commit incoming!

File: package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
     "test": "jest"
   },
   "devDependencies": {
-    "flow-bin": "^0.53.1",
-    "jest": "^20.0.4"
+    "flow-bin": "0.53.1",
```
Comment (ajaymathur) [inline package.json]: Is there a reason why the `^` from the flow version removed?

File: package.json [hunk 2]
```diff
@@ -21,7 +21,7 @@
...
+    "jest": "23.6.0"
```
Comment (ajaymathur) [inline package.json]: Can we use `^` dependency here instead of exact version just in case there are patches with bug fixes.

Comment (nem035) [reply package.json]: 👍

**Comment by jaybotttt** [comment]
Hey I have a cloud server on my phone without my knowledge  I was wondering
do you know how to take it off its cloud.so and there's also filed it
connects to that also .so filed if you know anything regarding that get
back to me

**Pull request merged**